src/data-access/repositories/test/mongo-test-database.ts

```diff
@@ -25,6 +25,10 @@ export class MongoTestDatabase {
     }
   }
 
+  public async teardownDatabase(): Promise<void> {
```

Comment:
Since this is a mongo test database class, then it would fine to call this method for `dropDatabase` to avoid confusion whether the database was dropped or cleared.